Additions of ecoinvent and IDEA mapping files to the FEDEFL

ashleyedelen opened the pull request:
These are drafts of the mapping files for IDEA and ecoinvent to the FEDEFL. They have undergone a preliminary internal review by ERG staff, but further review is recommended before finalizing.

ashleyedelen committed:
ecoinvent_FEDEFL mapping

This is the latest ecoinvent to FEDEFL mapping of ecoinvent version 3.6. This has undergone preliminary internal reviews, but further reviews and edits are recommended as this is an initial draft.

ashleyedelen committed:
IDEA_FEDEFL mapping

This is an initial mapping of the IDEA to FEDEFL mapping. This file has undergone a preliminary internal review, but further review and edits are recommended as this is a draft version.

WesIngwersen reviewed (changes requested):
See my comments requesting changes to these two files in the commits of your fork

ashleyedelen committed:
Delete ecoinvent.csv

wrong file uploaded

ashleyedelen committed:
Delete IDEA.csv

wrong file uploaded

ashleyedelen committed:
Correct file uploaded

ashleyedelen commented:
I uploaded the correct files. These files should include the correct naming.

the {} are from the IDEA source and are a copy of the original source

I believe the other issues have been corrected with the correct upload of the files.

WesIngwersen commented:
@ashleyedelen Please see the posted version of IDEAv2.2 in the main repo - the {} are not there
https://raw.githubusercontent.com/UNEP-Economy-Division/GLAD-ElementaryFlowResources/master/flowlists/IDEAv2.2.csv
And please confirm again that the flows you used match those in this list. thanks

Commit:
Delete IDEAv2.3toFEDEFLv1.0.3.csv

removing old mapping file

Commit:
Create ecoinventEFv3.6toFEDEFLv1.0.3.csv

uploaded ecoinventEFv3.6 mapping to FEDEFL

Commit:
Create IDEAv2.2toFEDEFLv1.0.3.csv

ashleyedelen committed:
Merge pull request #1 from hottleta/master

Updated mappings

hottleta commented:
I added the split flows to the bottom of the IDEA mapping for additional consideration. Depending on what we want to do, we can go ahead and map those individually.

WesIngwersen reviewed (changes requested):
@hottleta Please go ahead and propose actual mappings for those split flows rather than leaving the [potential split flows] in the file

Commit:
Update IDEAv2.2toFEDEFLv1.0.3.csv

updated possible split flows. diethylebenzene isomers and heptadecane seemed to map best to single flows

Commit:
Update IDEAv2.2toFEDEFLv1.0.3.csv

ashleyedelen committed:
Merge pull request #2 from hottleta/master

updated mapping for proposed split flows.

Commit:
correction in conversion factor to replace comma with decimal in ln 1015

WesIngwersen commented on mapping/IDEAv2.2toFEDEFLv1.0.3.csv:
```diff
@@ -0,0 +1,1022 @@
+IDEAv2.2,forest,fcd5e16b-e4c5-4c60-b045-042d30abd42d,Resources/land use/land occupation,m2year,=,1,Land use,1147e66c-db3e-303c-b7c7-89b006d98881,resource/ground/terrestrial/forest,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,mineral extraction site,CC0F5788-8059-4253-8498-D7B8D3CD3CD6,Resources/land use/land occupation,m2year,=,1,Land use,4b92c75e-e528-3f23-bc67-e92977682eb4,resource/ground/human-dominated/industrial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,dump yard,dd8c94cc-b5c2-4736-9dfc-668bce100891,Resources/land use/land occupation,m2year,=,1,Land use,4b92c75e-e528-3f23-bc67-e92977682eb4,resource/ground/human-dominated/industrial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,Iron manganese ore,1404b845-05d0-4d16-94f5-999775446c55,Resources/ground/Non-renewable material resources,kg,>,0.5,Iron ore,6fca43a7-13fb-315d-a184-8af7d40ff1d3,resource/ground,kg,Hottle,,8/28/2020
```
@kshobatake Please see mappings in this line forward where we split 1 IDEA flow across two FEDEFL flows. The conversion factors are also split in equal proportion to add to 1. If you have insight into better % composition data of the members of those groups in the IDEA flows, that could be used to improve the conversion factors

WesIngwersen commented on mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv:
```diff
@@ -0,0 +1,4103 @@
+SourceListName,SourceFlowName,SourceFlowUUID,SourceFlowContext,SourceUnit,MatchCondition,ConversionFactor,TargetFlowName,TargetFlowUUID,TargetFlowContext,TargetUnit,Mapper,Verifier,LastUpdated
+ecoinvent_v3.6,"1,4-Butanediol",38a622c6-f086-4763-a952-7c6b3b1c42ba,air/urban air close to ground,kg,=,1,"1,4-Butanediol",34d53001-84a2-3053-b887-32b891e334eb,emission/air/troposphere/urban/ground-level,kg,Edelen,Hottle,8/21/2020
```
@carl-vadenbo Please see the name used here. In the mapping files the SourceListName needs to match exactly the list name in the FlowList files, and the filenames of these files.  Please inform us of the standard name that you wish to use for this Ecoinvent flow list v3.6. thanks

carl-vadenbo replied:
@WesIngwersen Thanks for spotting this inconsistency. I haven't been able to locate where "ecoinvent_v3.6" is present, but let's go for "ecoinventEFv3.6" in any case. Please just let me know, should you need any further actions from my side to fix this, thanks!

WesIngwersen replied:
@carl-vadenbo what about just `ecoinventv3.6` ? i think that would be more consistent with the other list names like `IDEAv2.2` . the `EF` is implicit

carl-vadenbo replied:
@WesIngwersen sure, I'm fine with that, thanks!

WesIngwersen reviewed (changes requested):
Some fields are missing from the required format ..TargetListName and the 4 Memo fields at the end
See https://github.com/UNEP-Economy-Division/GLAD-ElementaryFlowResources/blob/master/formats/FlowMapping.md
which i just put up but was on the dropbox.
